src/test/java/teammates/test/cases/datatransfer/FeedbackQuestionAttributesTest.java

```diff
@@ -37,6 +40,201 @@ void setUpdatedAt(Instant updatedAt) {
+                .withShowRecipientNameTo(new ArrayList<>(participants))
+                .withShowResponseTo(new ArrayList<>(participants))
+                .withCreatedAt(createdAt)
+                .withUpdatedAt(updatedAt)
+                .withFeedbackQuestionId(feedbackQuestionId)
+                .build();
+
+        assertEquals(feedbackSession, feedbackQuestionAttributes.getFeedbackSessionName());
+        assertEquals(courseId, feedbackQuestionAttributes.getCourseId());
+        assertEquals(creatorEmail, feedbackQuestionAttributes.getCreatorEmail());
+        assertEquals(questionMetaData, feedbackQuestionAttributes.getQuestionMetaData());
+        assertEquals(questionDescription, feedbackQuestionAttributes.questionDescription);
+        assertEquals(questionNumber, feedbackQuestionAttributes.getQuestionNumber());
+        assertEquals(numOfEntities, feedbackQuestionAttributes.numberOfEntitiesToGiveFeedbackTo);
+        assertEquals(questionType, feedbackQuestionAttributes.getQuestionType());
+        assertEquals(giverType, feedbackQuestionAttributes.getGiverType());
+        assertEquals(recipientType, feedbackQuestionAttributes.getRecipientType());
+        assertEquals(participants, feedbackQuestionAttributes.showGiverNameTo);
+        assertEquals(participants, feedbackQuestionAttributes.showResponsesTo);
+        assertEquals(participants, feedbackQuestionAttributes.showRecipientNameTo);
+        assertEquals(createdAt, feedbackQuestionAttributes.getCreatedAt());
+        assertEquals(updatedAt, feedbackQuestionAttributes.getUpdatedAt());
+        assertEquals(feedbackQuestionId, feedbackQuestionAttributes.getFeedbackQuestionId());
+    }
+
+    @Test
+    public void testBuilderSanitizeForBuild() {
+
+        ______TS("sanitize whitespace");
+
+        Text contentWithWhitespaces = new Text(" content to be sanitized by removing leading/trailing whitespace ");
+        FeedbackQuestionAttributes feedbackQuestionAttributes = FeedbackQuestionAttributes.builder()
+                .withQuestionMetaData(new Text("test qn from teams->none."))
+                .withQuestionDescription(contentWithWhitespaces)
+                .build();
+
+        assertEquals(new Text("content to be sanitized by removing leading/trailing whitespace"),
+                feedbackQuestionAttributes.getQuestionDescription());
+
+        ______TS("sanitize code block");
+
+        Text codeblock = new Text("<code>System.out.println(\"Hello World\");</code>");
+        feedbackQuestionAttributes = FeedbackQuestionAttributes.builder()
+                .withQuestionMetaData(new Text("test qn from teams->none."))
+                .withQuestionDescription(codeblock)
+                .build();
+
+        assertEquals(new Text("<code>System.out.println(&#34;Hello World&#34;);</code>"),
+                feedbackQuestionAttributes.getQuestionDescription());
+
+        ______TS("sanitize superscript");
```

Comment:
Actually u don't need to write so many test cases for sanitizations. The purpose of this test should just test whether `sanitizeForRichText` is called or not. The additional test seems like we are testing whether `sanitizeForRichText` is doing a good job or not, which should be in the unit test of `SanitizationHelper` instead. But NVM, we can keep the test :)